packages/backend/src/services/validation.service.ts

```diff
@@ -351,7 +351,15 @@ export default class ValidationService {
       userQuizState.spamFlags > 0 &&
       this.openishStates.includes(quizAnswer.status)
     ) {
-      quizAnswer.status = "manual-review"
+      if (userQuizState.spamFlags >= 3) {
```

Comment:
This limit should be configurable just like we have the `max_spam_flags` in course. Maybe we could add this threshold as a new field to course table?

Also, I think we should flip comparison to be `if (userQuizState.spamFlags < 3)` because we want to send the answers with 3 or more spam flags directly to manual review.

Reply:
Made the limit configurable with `max_review_spam_flags` added to the course table. Migration also created. Also I flipped the comparison.